modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/noIntegration/EmbeddedCXFTestCase.java

```diff
@@ -55,11 +54,10 @@
  * @author alessio.soldano@jboss.com
  * @since 15-Apr-2013
  */
-@RunWith(Arquillian.class)
+@ExtendWith(ArquillianExtension.class)
+@EnabledOnJre(JRE.JAVA_17)
```

Comment:
Is there something in Junit5 to enable this test for Java17+ ?

Reply:
sure, there is an annotation we can use for that: `@EnabledForJreRange(min = JRE.JAVA_17)`